### ISSUE TITLE
Hyperparameters for reproducing unseen accuracy on cifar100

### ISSUE THREAD

**Issue by ShichangZh** [snap-stanford/orca#6]
Hello,

Thank you for the great work and for publishing the code!  On CIFAR-100, I followed the instruction and ran the following line three times, without making any changes to the code. The maximum unseen accuracy I got was 0.4114, 0.4148, and 0.3890 for these three runs. May I ask how can I get the 43.0 reported in the paper? Any hyperparameters I need to change?

```
python orca_cifar.py --dataset cifar100 --labeled-num 50 --labeled-ratio 0.5
```

**Comment by kleinzcy** [comment]
Hi, @ShichangZh 

Have you reimplement the results reported in the paper? We meet the same question as you. And I found that, setting mean_uncert as a fixed value (like 0.5) can get better results.

**Comment by ShichangZh** [comment]
No. I can't reproduce.
